orangecontrib/lookalike/widgets/owlookalike.py

```diff
@@ -330,10 +337,19 @@ def apply(self):
                           img_negh, self.orange_logo_item])
         widget.setPos(0, 60)
         self.scene.addItem(widget)
-
-        title = QGraphicsSimpleTextItem("I am {:.2f}% {}".format(
-            float(self.neighbors_model.index(self.neighbor_index, 1).data()),
```

Comment:
The distances cannot in general be used as a 'ratio' of similarity as was done here. You might (kind of) do that for cosine, pearson, abs pearson, ...) using `1 - distance`.

Reply:
We were aware of that. This was mean solely for demonstration purposes where we show the Celebrity Lookalike project to kids and they usually want to know 'how similar they are to a person'. This is impossible to quantify this way, so we cheated and create a false measure of similarity. Any other ideas welcome.
(p.s. We would be using this widget with cosine distance anyway...)

Reply:
@ajdapretnar @ales-erjavec Wanted to say that current implementation of the Lookalike widget changes the distance to percents. That means that we are showing how much different two persons are and not how similar they are. If we would want to tell you are so many percents similar to that person we would need to transform distance to similarity with `1-distance`. Currently, bigger value means that persons are more different and smaller than they are more similar. I think that Neighbor widget changed in this aspect from the last year.